docs/content/en/docs/yaml-crd-ref/task.md

```diff
@@ -0,0 +1,146 @@
+---
+title: KeptnTask
+description: Define a run of a KeptnTaskDefinition
+weight: 85
+---
+
+When using Keptn to run tasks for software
+that is deployed outside of Kubernetes,
+you must create the `KeptnTask` definition manually
+and modify it manually for each new run.
+Keptn automatically populates the `KeptnTask` resource
+for tasks that deploy software on Kubernetes.
+
+## Synopsis
+
+```yaml
+apiVersion: lifecycle.keptn.sh/v1alpha3
+kind: KeptnTask
+metadata:
+  name: <name-of-this-run>
+spec:
+  taskDefinition: <name-of-KeptnTaskDefinition resource>
+  context:
+    appName: "<name-of-KeptnApp-resource>"
+    appVersion: "1.0.0"
+    objectType: ""
+    taskType: ""
+    workloadName: "my-workload"
+    workloadVersion: "1.0.0"
+  parameters: <parameters to pass to job>
+  secureParameters: <secure parameters to pass to job>
+  checkType: ""
+  retries: <integer>
+  timeout: <duration-in-seconds>
+```
+
+## Fields
+
+* **apiVersion** -- API version being used.
+`
+* **kind** -- Resource type.
+   Must be set to `KeptnTask`
+
+* **metadata**
+  * **name** -- Unique name of this run of the task.
+    This name must be modified each time you run this `KeptnTask`,
+    so a common practice is to add a number to the end of the string
+    so you can increment the number for each run.
+    Names must comply with the
+    [Kubernetes Object Names and IDs](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names)
+    specification.
+* **spec** - Defines the speficication of this `KeptnTask` resource
+  * **workload** - Name of the
+      [KeptnWorkload](../crd-ref/lifecycle/v1alpha3/#keptnworkload)
+      resource for which the `KeptnTask` is being executed.
+
+  * **workloadVersion** - `KeptnWorkload` version
+      for which the `KeptnTask` is being executed.
+  * **appName** - Name of the
+      [KeptnApp](../yaml-crd-ref/app.md) resource
+      for which the `KeptnTask` is being executed.
+  * **appVersion** - Version of the `KeptnApp` resource
+      for which the `KeptnTask` is being executed.
+
```

Comment:
We do not support anymore these parameters in v1alpha3, see here https://github.com/keptn/lifecycle-toolkit/blob/main/lifecycle-operator/apis/lifecycle/v1alpha3/keptntask_types.go#L34

```suggestion
```